end_to_end_test/lib/enums.dart

```diff
@@ -10,21 +10,7 @@ import 'package:built_value/serializer.dart';
 
 part 'enums.g.dart';
 
-/// Example of how to use [EnumClass].
```

Comment:
Why do you remove the doc?

Reply:
There aren't supposed to be docs in the 'end_to_end_test' folder; the same file exists over in the 'example' folder with docs.

But, actually, I should add an example. Done.